pkg/issuer/acme/http/http_test.go

```diff
@@ -107,12 +107,15 @@ func TestReachabilityCustomDnsServers(t *testing.T) {
 		t.Fatalf("Failed to resolve %s: %v", u.Host, err)
 	}
 
+	dnsServerStarted := make(chan struct{})
 	dnsServerCalled := int32(0)
 
-	server := &dns.Server{Addr: "127.0.0.1:15353", Net: "udp"}
+	server := &dns.Server{Addr: "127.0.0.1:15353", Net: "udp", NotifyStartedFunc: func() { close(dnsServerStarted) }}
```

Comment:
We received a `read: connection refused` error when the test started before the DNS server was started.